lib/datasource.js

```diff
@@ -1656,6 +1656,8 @@ DataSource.prototype.discoverSchemas = function(tableName, options, cb) {
         dataPrecision: item.dataPrecision,
         dataScale: item.dataScale,
         nullable: item.nullable,
+        description: item.description,
+        default: item.default,
```

Comment:
Are you sure these two new items should be stored in db-specific metadata?

IMO, the `description` is not specific to the database being used and should be stored in `schema.properties[propName].description`, so that tooling like loopback-swagger or LB4' JSON/OpenAPI schema builders can use it.

I feel the same argument can be made for the `default` field too, but I am not so sure here.

@raymondfeng thoughts?